tools/manifest_files/patch.yaml

```diff
@@ -13,7 +13,7 @@ spec:
               fieldPath: metadata.name
         - name: OPERATOR_NAME
           value: bkop-bookkeeper-operator
-        image: pravega/bookkeeper-operator:0.1.2
+        image: pravega/bookkeeper-operator:0.1.3
```

Comment:
This change is not required since the version mentioned here is anyways going to be patched by the following line in the upgrade script
https://github.com/pravega/bookkeeper-operator/blob/master/tools/operatorUpgrade.sh#L51